include/ginkgo/core/distributed/preconditioner/schwarz.hpp

```diff
@@ -89,6 +91,19 @@ class Schwarz
          */
         std::shared_ptr<const LinOp> GKO_FACTORY_PARAMETER_SCALAR(
             generated_local_solver, nullptr);
+
+        /**
+         * Operator factory to generate the triplet (prolong_op, coarse_op,
+         * restrict_op).
+         */
+        std::shared_ptr<const LinOpFactory> GKO_DEFERRED_FACTORY_PARAMETER(
+            galerkin_ops);
```

Comment:
I'm not too convinced of the name and the doc. At least to me, `galerkin_ops` is too unspecific, maybe something like `coarse_level_factory` would be better suited. Also the doc should contain that this will be used to create a coarse level system, maybe 
```
Operator factory to generate the coarse system `A_c = R * A * P`. 
```